### PR TITLE
Added new CI lane for Prime/Alpha build testing

### PR DESCRIPTION
**Pull request by sbulage**
Fixes: #183

### CODE REVIEW

File: .github/workflows/rm-prime_2.8_and_2.9.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      azure_private_pwd: ${{ secrets.AZURE_PRIVATE_PWD }}
+      rsa_private_key_qa: ${{ secrets.RSA_PRIVATE_KEY_QA }}
+      rsa_public_key_qa: ${{ secrets.RSA_PUBLIC_KEY_QA }}
+      qase_api_token: ${{ secrets.QASE_API_TOKEN }}
+    with:
+      test_description: "CI/Manual - UI - Deployment test with Standard K3s"
+      cluster_name: cluster-k3s
+      # WARNING, VALUES BELOW ARE HARDCODED FOR RUNS SCHEDULED BY CRON
+      destroy_runner: ${{ github.event_name == 'schedule' && true || inputs.destroy_runner }}
+      upstream_cluster_version: ${{ inputs.upstream_cluster_version || 'v1.28.8+k3s1' }}
```
Comment (thehejik) [inline .github/workflows/rm-prime_2.8_and_2.9.yaml:55]: use the new version here which will be used for scheduled job.

File: .github/workflows/rm-prime_2.8_and_2.9.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      rancher_version: ${{ inputs.rancher_version || 'latest/devel/2.9' }}
```
Comment (thehejik) [inline .github/workflows/rm-prime_2.8_and_2.9.yaml:56]: dtto... `prime/2.9.1`

File: .github/workflows/rm-prime_2.8_and_2.9.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        description: K3s upstream cluster version where to install Rancher
+        default: 'v1.30.3+k3s1'
+        type: string
+        required: true
+      grep_test_by_tag:
+        description: Grep tags. For multiple selection separate with spaces. Keep always @login
+        required: false
+        type: string
+        default: '@login @p0 @p1 @rbac'
+
```
Comment (thehejik) [inline .github/workflows/rm-prime_2.8_and_2.9.yaml:31]: the schedule[cron] here is mission on purpose?

File: .github/workflows/rm-prime_2.8_and_2.9.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# This workflow calls the master E2E workflow with custom variables
+name: UI-RM_PRIME_2.8_AND_2.9
```
Comment (thehejik) [inline .github/workflows/rm-prime_2.8_and_2.9.yaml:2]: what about `UI-RM_Prime`?

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 1 above)
Comment (thehejik) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml:55]: well, the string after `||` is used only for scheduled jobs so you can remove it and keep just inputs there

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 2 above)
Comment (thehejik) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml:56]: valid only for scheduled...remove if you dont plan to use cron

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 1 above)
Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml:55]: Nice catch :smiley:

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 2 above)
Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml:56]: Nice catch :smiley: Updated locally, Pushing changes.

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 4 above)
Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml:2]: It will indicate UI test cases for Rancher Manager Prime.

Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml:2]: No, this workflow will consist of Prime RC's/Alpha's as well released one too. Can you please suggest any other name?

Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml:2]: How about: `UI-RM_PRIME_TESTING` ?

File: .github/workflows/rm-prime_2.8_and_2.9.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        default: true
+        type: boolean
+      rancher_version:
+        description: Rancher version channel/version/head_version latest/2.9.x[-rc1], prime/2.9.x, prime/devel/2.9, alpha/2.9.0-alpha1
+        default: prime/2.9.1
+        type: string
+        required: true
+      upstream_cluster_version:
+        description: K3s upstream cluster version where to install Rancher
+        default: 'v1.30.3+k3s1'
```
Comment (mmartin24) [inline .github/workflows/rm-prime_2.8_and_2.9.yaml:23]: do we want also `1.28` to be 2.8 compatible or?

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 4 above)
Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml:2]: How about this: `UI-RM_Prime_Alpha_RC`

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 1 above)
Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml]: Updated.

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 2 above)
Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml]: Updated.

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 4 above)
Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml]: Updated with new suggested names.

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 5 above)
Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml]: Yes, I will consider K8s 1.28 version support for both versions 2.8 and 2.9.

File: .github/workflows/ui-rm_prime_alpha_rc.yaml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+      qase_run_id:
+        description: Qase run ID where the results will be reported. (Please update with custom run ID.)
+        default: none
+        type: string
+      destroy_runner:
+        description: Destroy the auto-generated self-hosted runner
+        default: true
+        type: boolean
+      rancher_version:
+        description: Rancher version channel/version/head_version latest/2.9.x[-rc1], prime/2.9.x, prime/devel/2.9, alpha/2.9.0-alpha1
```
Comment (mmartin24) [inline .github/workflows/ui-rm_prime_alpha_rc.yaml:17]: Non blocker, but perhaps to consider for further iterations 
`alpha/2.9.0-alpha1` to `alpha/2.x.x-alphax`

**Review by mmartin24 — APPROVED** [review]
Small suggestion for the furture but LGTM

File: .github/workflows/ui-rm_prime_alpha_rc.yaml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      azure_private_user: ${{ secrets.AZURE_PRIVATE_USER }}
+      azure_private_pwd: ${{ secrets.AZURE_PRIVATE_PWD }}
+      rsa_private_key_qa: ${{ secrets.RSA_PRIVATE_KEY_QA }}
+      rsa_public_key_qa: ${{ secrets.RSA_PUBLIC_KEY_QA }}
+      qase_api_token: ${{ secrets.QASE_API_TOKEN }}
+    with:
+      test_description: "CI/Manual - UI - Deployment test with Standard K3s"
+      cluster_name: cluster-k3s
+      # WARNING, VALUES BELOW ARE HARDCODED FOR RUNS SCHEDULED BY CRON
+      destroy_runner: ${{ github.event_name == 'schedule' && true || inputs.destroy_runner }}
```
Comment (thehejik) [inline .github/workflows/ui-rm_prime_alpha_rc.yaml:54]:
```suggestion
      destroy_runner: ${{ inputs.destroy_runner }}
```

**Review by thehejik — APPROVED** [review]

File: .github/workflows/rm-prime_2.8_and_2.9.yaml (hunk 5 above)
Comment (sbulage) [reply .github/workflows/rm-prime_2.8_and_2.9.yaml]: Updated it.

File: .github/workflows/ui-rm_prime_alpha_rc.yaml (hunk 7 above)
Comment (sbulage) [reply .github/workflows/ui-rm_prime_alpha_rc.yaml]: Okay, I will update it.

File: .github/workflows/ui-rm_prime_alpha_rc.yaml (hunk 6 above)
Comment (sbulage) [reply .github/workflows/ui-rm_prime_alpha_rc.yaml]: Updated it.

**Pull request merged**